spec/features/movies/search_spec.rb

```diff
@@ -47,4 +47,12 @@
     expect(page).to have_button('Find Movies')
     expect(page).to have_button('Find Top Rated Movies')
   end
+
+  it 'each title on movies page links to the movies detail page' do
+    fill_in :search, with: @query
+    click_button('Find Movies')
+    
+    expect(page).to have_link("Perfume: The Story of a Murderer", :href=>'/movies/1427')
```

Comment:
I really appreciate the testing of the link functionality and not that it just says what it should. Current path testing takes us more space so this is great!